pytest_selenium/drivers/cloud.py

```diff
@@ -30,13 +30,14 @@ def config(self):
         config.read([name, os.path.join(os.path.expanduser('~'), name)])
         return config
 
-    def get_credential(self, key, env):
-        try:
-            value = self.config.get('credentials', key)
-        except (configparser.NoSectionError,
-                configparser.NoOptionError,
-                KeyError):
-            value = os.getenv(env)
-        if not value:
-            raise MissingCloudCredentialError(self.name, key, env)
-        return value
+    def get_credential(self, key, envs):
+        for env in envs:
+            try:
+                value = self.config.get('credentials', key)
+            except (configparser.NoSectionError,
+                    configparser.NoOptionError,
+                    KeyError):
+                value = os.getenv(env)
+            if value:
+                return value
+        raise MissingCloudCredentialError(self.name, key, envs)
```

Comment:
It looks like we should reword the message for this custom exception to handle the list of environment variable names.